contract/src/start-game1-proposal.js

```diff
@@ -18,75 +18,89 @@ const CENT = IST_UNIT / 100n;
 /**
  * Make a storage node for auxilliary data for a value on the board.
  *
- * @param {ERef<StorageNode>} chainStorage
- * @param {string} boardId
+ * @param {ERef<StorageNode>} chainStorage - Reference to the chain storage.
+ * @param {string} boardId - Identifier for the board.
  */
 const makeBoardAuxNode = async (chainStorage, boardId) => {
+  // Create a child node under BOARD_AUX in chain storage.
   const boardAux = E(chainStorage).makeChildNode(BOARD_AUX);
```

Comment:
again, does "Create a child node" communicate something that `makeChildNode` does not?

"chain storage" is, if anything, less clear than `chainStorage`.

And "under BOARD_AUX" is somewhat ambiguous, since the value of `BOARD_AUX` is the string `boardAux`.

Reply:
Understood